index.html

```diff
@@ -856,6 +856,22 @@ <h3>
           </li>
         </ol>
       </li>
+      <li>If the session's [=next wallet behavior=] is set:
```

Comment:
I wonder if we need this these steps at all?... this should really be covered by #306.